### DISCUSSION TITLE
BalenaOS Installation | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#362] [Q&A]
# BalenaOS Installation | Seeed Studio Wiki

Getting Started with balenaOS on ODYSSEY-X86

https://wiki.seeedstudio.com/Quantum-Mini-Linux-Development-Kit/

**Comment by Afshin-atelier** [comment]
Hello, is the quantom mini linux dev kit good for runing yolov5?

**Reply by MatthewJeffson** [reply]
Hi! We haven't test it yet. I will be really grateful if you can test it. :D

**Reply by Afshin-atelier** [reply]
I dont have the board, i want to make sure before buying it.
But thanks for your reply.
